### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by coolboole** [dunbarcyber/cyphondock#18]
Hi,

So I'm new to using Docker so chances are I've probably done something stupid so apologies in advance.

I have attempted to install Cyphon on a Ubuntu 16.04 LTS Server Virtualbox VM.

I installed PostgreSQL, RabbitMQ and Logstash and Docker via Aptitude.

I then followed the instructions [here](https://cyphon.readthedocs.io/en/latest/getting-started/install-cyphon.html), so far so good.

Finally I pulled down Cyphon and Cyclops from Docker successfully.

So I go to run Cyphon to test that it works. I ran the command `sudo docker run -it -p 8000:8000 -p 5601:5601 -p 15672:15672 dunbar/cyphon` allowing access to the ports specified [here](https://cyphon.readthedocs.io/en/latest/getting-started/start-docker.html#ports).

I then get a problem to do with port 5432 which I think is to do with PostgreSQL. I checked that it was running and port it was using and it seems okay.

I can't seem to find the problem, I've attached the [error messages](https://github.com/dunbarcyber/cyphondock/files/1051985/Error.Message.-.Cyphon.txt) to this post in a txt file.

Thanks,

Coolboole

**Comment by lhadjchikh** [comment]
There are a couple approaches you could take to this. One is to use the Docker Compose files, which would handle PostgreSQL and all the related services (you wouldn't need to install them with Aptitude; Docker would download them as images and run them in containers, which would already be configured to talk to Cyphon). You can read about that option [here](http://cyphon.readthedocs.io/en/latest/getting-started/start-docker.html).

Alternatively, you could open port 5432 on your Cyphon container so it can connect to PostgreSQL and port 5672 so it can connect to RabbitMQ (assuming both are running on localhost). No need to expose ports 5601 (Kibana) and 15672 (RabbitMQ management) for the Cyphon container; they don't need to communicate with Cyphon. So your Docker command would look like this:
```
sudo docker run -it -p 8000:8000 -p 5432:5432 -p 5672:5672 dunbar/cyphon
```
DISCLAIMER: I haven't actually tested this yet! However, we're planning to work on this kind of deployment in the near future, so we'll figure out what works.

We've just released [Cyphon 1.2](https://github.com/dunbarcyber/cyphon/releases/tag/1.2.0), which does away with the need for a separate Cyclops container. You can read about it [here](https://www.cyphon.io/blog/2017/6/5/cyphon-release-12).

Let us know if you still have issues getting things up and running!

**Comment by lhadjchikh** [comment]
Closing this due to inactivity, but feel free to reopen if you're still having issues.
